README.md

```diff
@@ -155,6 +147,8 @@ To explore any operator samples built using the operator-sdk, see the [operator-
 
 See [CONTRIBUTING][contrib] for details on submitting patches and the contribution workflow.
 
+See the [proposal docs][proposals_docs] and issues for ongoing or planned work.
```

Comment:
This line seemed worth keeping for discoverability. Does it make sense to include it in the Contributing section?

Reply:
Yeah I think it's worth pointing out the proposals so this is fine.